framework/src/mesh/MooseMesh.C

```diff
@@ -2832,7 +2915,7 @@ MooseMesh::getSubdomainInterfaceBoundaryIds(SubdomainID subdomain_id) const
 }
 
 std::set<SubdomainID>
-MooseMesh::getBoundaryConnectedBlocks(const BoundaryID bid) const
+MooseMesh::getBoundaryConnectedBlocks(BoundaryID bid) const
```

Comment:
The problem is that the argument is passed in by value, so const here does not make sense. `bid` is not going to be modified by the function anyway. I think this is actually a good argument for c++ developers. Removing this is also making it consistent with several other places.

Reply:
`const` absolutely still makes sense for pass-by-value. It's the same reason you write `const` for any constructed object in a code-block. It represents a contract that there's no reason to modify the object. I agree that it will have no affect on the caller of this method since it's pass-by-value, but this doesn't mean there isn't value. I refer you here: https://stackoverflow.com/a/117557/4493669

Reply:
> Removing this is also making it consistent with several other places.

If anything those other places should be updated to be `const` (in the .C file), instead of the other way around

Reply:
This won't be the the hill that I die on; I'll accept an acknowledgement that there's value in the suggested changes :smile:

Reply:
Just undo this change

Reply:
Ok, I will revert this change.